### DISCUSSION TITLE
I cannot find the FedSM code

### DISCUSSION THREAD

**Discussion by FibonacciZ** [NVIDIA/NVFlare#1391] [General]
The paper "Closing the Generalization Gap of Cross-silo Federated Medical Image Segmentation" shows that the source code saved in https://github.com/NVIDIA/NVFlare/examples/FedSM. But I cannot find it.

**Comment by YuanTingHsieh** [comment]
@ZiyueXu77 can you help answer this question, is the code in draft PR?

**Comment by ZiyueXu77** [comment]
Hi @FibonacciZ, the code is currently under review on this PR https://github.com/NVIDIA/NVFlare/pull/562, some further updates will be made to Readme, but function-wise it is ready

**Comment by holgerroth** [comment]
FedSM code has been merged https://github.com/NVIDIA/NVFlare/tree/dev/research/fed-sm
